app.js

```diff
@@ -1,25 +1,36 @@
-const express = require('express');
-
-const hbs = require('hbs');
-const path = require('path');
-const PunkAPIWrapper = require('punkapi-javascript-wrapper');
-
-const app = express();
-const punkAPI = new PunkAPIWrapper();
-
-app.set('view engine', 'hbs');
-app.set('views', path.join(__dirname, 'views'));
-
-app.use(express.static(path.join(__dirname, 'public')));
-
-// Register the location for handlebars partials here:
-
-// ...
-
-// Add the route handlers here:
-
+// Importaciones
+const express = require('express')
+const app = express()
+const hbs = require('hbs')
+const PunkAPIWrapper = require('punkapi-javascript-wrapper')
+const punkAPI = new PunkAPIWrapper()
+require('dotenv').config()
+
+// 2.- Middlewares
+app.use(express.static("public"))
+app.set('views', __dirname + '/views')
+app.set('view engine', 'hbs')
+hbs.registerPartials(__dirname + '/views/partials')
+
+// 3.- Rutas
+app.get("/random-beer", (req, res) => {
+    const randomBeers = punkAPI.getRandom()
```

Comment:
Podemos omitir esta variable ya que el .then, recibe como parametro el resultado de nuestra promesa, podemos concatenar los metods de la siguente manera es una alternativa
```suggestion
punkAPI
    .getRandom() // .getRandom() es el método proporcionado por punkAPI
    .then((responseFromApi) => {
      // console.log("Response is:",  responseFromApi);
      res.render('beers/random-beer.hbs', { beers: responseFromApi });
    })
    ```